Automattic/jetpack
# E2E Tests: Disable Allure reports generation

For quite some time we stopped generating allure reports on heroku. So the step that is being disabled is useless and only takes processing time.

#### Jetpack product discussion


n/a
#### Does this pull request change what data or activity we track or use?


n/a

#### Testing instructions:






* Not much to test. 
* Make sure that E2E actions does not try to generate allure reports

#### Proposed changelog entry for your changes:


* n/a
